data/json/monsters/zed-pupating.json

```
@@ -61,6 +61,74 @@
       ]
     }
   },
+  {
+    "id": "mon_brute_pupa_decoy",
+    "type": "MONSTER",
+    "name": { "str": "pupating brute" },
+    "description": "This muscular human corpse is wrapped in sticky black fibers that cover everything neck down.  Beneath the wrapping there is strange rhythmic movement grotesque to behold.",
+    "copy-from": "mon_zombie_brute",
+    "delete": { "categories": [ "CLASSIC" ] },
+    "speed": 95,
+    "armor_bash": 8,
+    "armor_cut": 6,
+    "armor_bullet": 6,
+    "regenerates": 10,
+    "upgrades": { "half_life": 21, "into_group": "GROUP_BRUTE_PUPA" },
+    "extend": { "flags": [ "SLUDGETRAIL", "SLUDGEPROOF" ] }
+  },
+  {
+    "id": "mon_brute_pupa",
+    "type": "MONSTER",
+    "name": { "str": "pupating brute" },
+    "description": "This muscular human corpse is wrapped in sticky black fibers that cover everything neck down.  Beneath the wrapping there is strange rhythmic movement grotesque to behold.",
+    "copy-from": "mon_brute_pupa_decoy",
+    "extend": {
+      "special_attacks": [
+        {
+          "type": "spell",
+          "spell_data": { "id": "big_raptor_spawn", "hit_self": true },
+          "cooldown": 50,
+          "monster_message": "The pupating brute bursts, and gore-smeared winged beasts climb out of the corpse!"
+        }
+      ]
+    }
+  },
+  {
+    "id": "mon_hulk_pupa_decoy",
+    "type": "MONSTER",
+    "name": { "str": "hive hulk" },
+    "description": "This gigantic corpse is wrapped in sticky black fibers that cover everything neck down.  Beneath the wrapping there is strange rhythmic movement grotesque to behold.",
+    "copy-from": "mon_zombie_hulk",
+    "delete": { "categories": [ "CLASSIC" ] },
+    "armor_bash": 8,
+    "armor_cut": 6,
+    "armor_bullet": 6,
+    "melee_skill": 4,
+    "melee_dice": 3,
+    "melee_dice_sides": 7,
+    "hp": 300,
+    "speed": 85,
+    "special_attacks": [ { "type": "SMASH", "cooldown": 30 } ],
+    "regenerates": 10,
+    "extend": { "flags": [ "SLUDGETRAIL", "SLUDGEPROOF" ] }
+  },
+  {
+    "id": "mon_hulk_pupa",
+    "type": "MONSTER",
+    "name": { "str": "hive hulk" },
+    "description": "This gigantic corpse is wrapped in sticky black fibers that cover everything neck down.  Beneath the wrapping there is strange rhythmic movement grotesque to behold.",
+    "copy-from": "mon_hulk_pupa_decoy",
```

```suggestion
    "copy-from": "mon_hulk_pupa_decoy",
```
I would suggest trying to break the circular json by not using copy-from here and adding the needed lines from where you are copying from.  If that doesn't work I'd look at whether you have created in the monster evolution's a loop where a monster evolves and copies from a monster that only exists via copy from.  That's probably a poor description but it's the best description I can think of for some tricky evolve messups that I've made previously.

---

I uncopied them and half the errors were gone. Thanks!